frontend/src/app/modules/common/header/quick-add-menu.sass

```diff
@@ -2,33 +2,20 @@
   @include breakpoint(680px down)
     display: none
 
-  &--button
-    display: flex
-    justify-content: center
-    align-items: center
-    position: relative
-    color: var(--button--alt-highlight-font-color)
-    font-size: 0.8rem
-
-    > *
-      z-index: 1
-
-    &::after
-      display: block
-      background: var(--button--alt-highlight-background-color)
-      border-width: 2px
-      border-style: solid
-      border-color: var(--button--alt-highlight-background-color)
-      border-radius: 50%
-      padding: 0.25rem
-      width: 26px
-      height: 26px
-      content: ''
-      position: absolute
+  .op-quick-add-menu--button
```

Comment:
this increases the specificity of the selector by one class, I'd really prefer if we stick to `&--button`. Is there a psecific reason for this change?